tests/test_nwm22_point_loading.py

```diff
@@ -26,11 +34,17 @@ def test_point_loading():
         True,
```

Comment:
@samlamont kind of nit picky, but I think adding the argument name before the argument is preferable (num_chunks=100 vs  just 100) esp. when there are lots of arguments.  Otherwise it is hard to tell what is going on.